### ISSUE TITLE
yapf makes changed to code inside a region marked with '# yapf: disable' and '# yapf: enable'

### ISSUE THREAD

**Issue by kkheller** [google/yapf#146]
here is a test input file, disable_test.py:

```
class MyClass:
    def some_getter(self):
        print('some_getter')
        return 1

    def some_getter_02(self):
        print('some_getter')
        return 1

    def _fail_setter(self, value):
        print('refusing to set value')

    TEST_AB    = property(some_getter,   _fail_setter)
    TEST_ABCD  = property(some_getter,   _fail_setter)
    TEST_CDEF  = property(some_getter_02,_fail_setter)

    # yapf: disable
    TYPE_AB    = property(some_getter,   _fail_setter)
    TYPE_ABCD  = property(some_getter,   _fail_setter)
    TYPE_CDEF  = property(some_getter_02,_fail_setter)
    # yapf: enable
```

here is my command line:
```
PYTHONPATH=~/Downloads/yapf python ~/Downloads/yapf/yapf  -i disable_test.py 
```

I expect that yapf will make a lot of changes to the whitespace on the lines with TEST_AB, TEST_ABCD, and TEST_CDEF.  Indeed, those changes are made by yapf. Expectations met.

However, i also expect that nothing would change on the lines with TYPE_AB, TYPE_ABCD, and TYPE_CDEF.

Instead of getting what i expect on the disabled section, I get this:

```
    TYPE_AB    = property(some_getter,   _fail_setter)
    TYPE_ABCD  = property(some_getter,   _fail_setter)
    TYPE_CDEF  = property(some_getter_02, _fail_setter)
```

(no changes were made to the first 2 lines. one space char was inserted after the comma on that third line.)

Maybe this outcome is expected?  If so, I can live with that :)

It seems, however, that using the disable-region feature should mean that none of the lines in that region would change.

I am using e72600e7 from master.  The same behavior is observed whether I invoke yapf using python 2.7 or python 3.4.  Thanks!

**Comment by gwelymernans** [comment]
In general, we try not to modify anything in those regions. In practice, we sometimes have to. For instance, if the code around the disable region is dedented or indented, then the code within the disable regions needs to be similarly modified. Because of that, all of the code within the disable region runs through the formatter. It just instructs the formatter to not mess with the original formatting. You've stumbled across one instance where the formatter isn't ignoring the original formatting as it should.

TL;DR It looks like a bug. :-)
